Doc/reference/datamodel.rst

```diff
@@ -885,7 +885,7 @@ Predefined (writable) attributes:
    :attr:`__name__`
       The module's name.
 
-   :attr:`__doc__`
+   :attr:`~definition.__doc__`
```

Comment:
I'd prefer to fix these warnings by creating a single canonical table in the datamodel docs, and removing duplicated documentation elsewhere: see https://github.com/python/cpython/issues/101100#issuecomment-1852929714 (and see #112781, where I did a similar thing for code object attributes)